src/client/components/app/Transactions/components/TxTokenInput.tsx

```diff
@@ -278,40 +278,42 @@ export const TxTokenInput: FC<TxTokenInputProps> = ({
         </CSSTransition>
       )}
 
-      <TokenInfo>
-        <TokenSelector onClick={listItems?.length > 1 ? openSearchList : undefined}>
-          <TokenIconContainer>
-            <TokenIcon icon={selectedItem.icon} symbol={selectedItem.label} size="big" />
-            {listItems?.length > 1 && <TokenListIcon Component={ChevronRightIcon} />}
-          </TokenIconContainer>
-          <TokenName>{selectedItem.label}</TokenName>
-        </TokenSelector>
-
-        <TokenData>
-          <AmountTitle>{inputText || t('components.transaction.token-input.balance')}</AmountTitle>
-          <StyledAmountInput
-            value={amount}
-            onChange={onAmountChange ? (e) => onAmountChange(e.target.value) : undefined}
-            placeholder={loading ? loadingText : '00000000.00'}
-            readOnly={readOnly}
-            error={inputError}
-            type="number"
-          />
-          <TokenExtras>
-            {amountValue && <StyledText>≈ {formatUsd(!loading && !inputError ? amountValue : '0')}</StyledText>}
-            {maxAmount && (
-              <StyledButton onClick={onAmountChange ? () => onAmountChange(maxAmount) : undefined}>
-                {maxLabel}
-              </StyledButton>
-            )}
-            {yieldPercent && (
-              <StyledText>
-                {t('components.transaction.token-input.yield')} <ContrastText>{yieldPercent}</ContrastText>
-              </StyledText>
-            )}
-          </TokenExtras>
-        </TokenData>
-      </TokenInfo>
+      <>
```

Comment:
Are we sure these are needed now? Usually fragment is used when we want to group a list of children without adding extra nodes to the DOM.

Reply:
That makes sense, just wondering if there's a better way. I guess we can go with that for now.